contracts/solc-0.8.8/Manager.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+// solhint-disable-next-line
+pragma solidity 0.8.8;
+
+import "./IManager.sol";
+import "./IController.sol";
+
+contract Manager is IManager {
```

Comment:
Note that the following functions were dropped from the original [Manager](https://github.com/livepeer/protocol/blob/streamflow/contracts/Manager.sol):

- whenSystemNotPaused
- whenSystemPaused

because they are not needed in BridgeMinter.